idaes/models_extra/power_generation/unit_models/soc_submodels/common.py

```diff
@@ -477,7 +477,7 @@ def __call__(self, possible_subset):
         "G": 0.0,
         "H": 0.0,
     },
-    "O^2-": {  # Chosen to match O2 coeffs besides enthalpy and entropy of formation
+    "O^2-": {  # Chosen to match O2 coeffs
```

Comment:
I didn't intend to commit this, but this correction to the comment is correct for the version of the model in the repo, so let's just leave it in.